db.py

```diff
@@ -607,7 +619,7 @@ def get_todos(username: str) -> List[Dict]:
             "route": "/public/team/registration"
         },
         {  # player
-            "completed": len(players) == 5,
+            "completed": len(players) >= 5,
```

Comment:
don't change this, a team with more than 5 members cannot compete in the competition and thus should not be marked as "registration complete" as something is clearly wrong